topics/statistics/tutorials/RNN/tutorial.md

```diff
@@ -211,28 +211,53 @@ cost function via backpropagation.
 
 # Get data
 
+Let's start with a fresh history.
+
+> ### {% icon hands_on %} Hands-on: Create history
+>
+> 1. Make sure you have an empty analysis history.
+>
+>    {% snippet snippets/create_new_history.md %}
+>
+> 2. **Rename your history** to make it easy to recognize
+>
+>    > ### {% icon tip %} Rename a history
+>    >
+>    > * Click on the title of the history (by default the title is `Unnamed history`)
+>    >
+>    >   ![Renaming history](../../../../shared/images/rename_history.png)
+>    >
+>    > * Type `Galaxy Introduction` as the name
+>    > * Press <kbd>Enter</kbd>
+>    >
+>    {: .tip}
```

Comment:
```suggestion
>    {% snippet snippets/rename_history.md %}
```

if the name is important, we can use the templating features of snippets to do that.

Reply:
Thx, merging this now.